R/plots.R

```diff
@@ -510,6 +510,14 @@ plot_regular_grid <- function(x, metric = NULL, eval_time = NULL, ...) {
   dat <- paste_param_by(dat)
   dat <- dat %>% dplyr::filter(!is.na(mean))
   dat <- filter_plot_eval_time(dat, eval_time)
+
+  if (all(vctrs::vec_count(dat$.metric)$count == 1)) {
+    cli::cli_abort(
```

Comment:
Can we avoid 

> Error in `plot_regular_grid()`:

by passing the calling environment through?

Reply:
sure thing, i was on the fence, whether to do it now or wait for the {cli} push

Reply:
done in [`01c6ded` (#801)](https://github.com/tidymodels/tune/pull/801/commits/01c6ded9b770bd7299dda048f7cc1d7229f0885a)